bundles/core/org.eclipse.smarthome.core/META-INF/MANIFEST.MF

```diff
@@ -30,7 +30,8 @@ Export-Package:
  org.eclipse.smarthome.core.service,
  org.eclipse.smarthome.core.storage,
  org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.core.types.util
+ org.eclipse.smarthome.core.types.util,
+ org.eclipse.smarthome.core.util

```

Comment:
This package should also be mentioned in the documentation as "allowed for bindings" and be included as an import in the archetype, considering it would be a perfect place for those few collection utilities that would finally make guava unnecessary. Just mentioning - this IMHO doesn't need to be in this PR!